configure

```diff
@@ -294,15 +294,16 @@ parser.add_option('--without-perfctr',
     dest='without_perfctr',
     help='build without performance counters')
 
+# Dummy option for backwards compatibility
 parser.add_option('--with-snapshot',
     action='store_true',
-    dest='with_snapshot',
+    dest='unused_with_snapshot',
     help=optparse.SUPPRESS_HELP)
 
-# Dummy option for backwards compatibility.
 parser.add_option('--without-snapshot',
-    action='store_true',
-    dest='unused_without_snapshot',
+    action='store_false',
+    dest='with_snapshot',
```

Comment:
I think it would be clearer / more idiomatic if you named it `'without_snapshot'` with `action='store_false'` (and you can drop the `default=True` in that case.)